.travis.yml

```diff
@@ -6,7 +6,9 @@ before_script:
   - "sh -e /etc/init.d/xvfb start"
   - "npm install -g bower karma grunt-cli"
   - "bower install"
-script: grunt test
+script:
+  - grunt jshint
+  - grunt test
```

Comment:
it's a good idea to also put it also in the server tasks. however I think travis should also run it.